flaskbb/plugins/utils.py

```diff
@@ -10,12 +10,12 @@
     :license: BSD, see LICENSE for more details.
```

Comment:
Thought I'd need changes here and ended up running black on it. I can back out these changes if we want.

Reply:
That's ok.